### DISCUSSION TITLE
Welcome to awesome-chatgpt Discussions!

### DISCUSSION THREAD

**Discussion by jordn** [humanloop/awesome-chatgpt#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Anujchobe** [comment]
How can we contribute to the code if we wanted to as this is not an open source project

**Reply by wijjj** [reply]
This. Looking for a (partial) replication segment.

**Comment by papajuju** [comment]
What would you rather be or a wasp.?

**Comment by decoOce** [comment]
I thought this is not open source?

**Comment by iush12** [comment]
Nice

**Comment by Giftedvon360** [comment]
You're welcome

**Comment by maykingmagic** [comment]
Hello, hope y'all are having a great day.

**Comment by i-mangithub** [comment]
Hello am new here don't really understand how things work here. Please i need to be enlighten

**Comment by jordn** [comment]
excellent discussions, all. :)

**Comment by chijioke07** [comment]
Great

**Comment by Mehi-Skid** [comment]
Bug report: it refuses to tell me the ingredients to heroin

**Comment by CoryfromCordesville** [comment]
Why so leftist?
